process/TI-Gives+Gets.md

```diff
@@ -0,0 +1,52 @@
+# "Gives and Gets" for OpenSSF Technical Initiatives (TI)
+The OpenSSF has a large community of contributors and efforts that span the broad spectrum of open source security interests.  The Technical Initiatives (TIs) of the foundation are where our members collaborate and help craft unique solutions to addressing improving the security of the open source ecosystem.
```

Comment:
Do we want to also highlight that his gives and gets is a bit vague on details and how someone should approach understanding more?

> May post project updates and tutorials to the OpenSSF blog.

How many? How often?

I know we don't have an answer to that but we might want to highlight that this is access to these benefits but the details will depend on time, scope, staffing, etc.

Reply:
I would suggest to make the guidance as broad as possible.  I'm not sure telling someone they can post 4 blogs a year really adds anything.  If folks have work output to share that adds value, they should get it shared; if they don't or on working on a long-running project, that's ok too.  But that's just my opinion.